tautulli/docker-compose.yml

```diff
@@ -0,0 +1,14 @@
+version: "3.7"
+
+services:
+  app_proxy:
+    environment:
+      APP_HOST: tautulli
+      APP_PORT: 8181
+
+  server:
+    image: lscr.io/linuxserver/tautulli:develop-5a8797a6-ls247@sha256:88a928a7cc4498fac36774f944204a2c0d8c23998e8b4604f4b8220447bf12dd
+    container_name: tautulli
+    volumes:
+      - ${APP_DATA_DIR}/data/config:/config
+    restart: on-failure
```

Comment:
Can you also please add the usual linuxserver `environment` section to run the container as 1000:1000?

```
environment:
      - PUID=1000
      - PGID=1000
 ```